pages/blog/documenting-sqs-event-driven-apis-with-asyncapi.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Documenting SQS event-driven APIs with AsyncAPI
+date: 2022-08-18T12:40:20-03:00
```

Comment:
@amuroBosetti let's release it on Thursday

```suggestion
date: 2022-09-22T06:00:00-00:00
```

Reply:
Done! I didn't mean to dismiss @alequetzalli by committing this though :disappointed: